src/main/java/grondag/canvas/shader/GlProgram.java

```diff
@@ -247,6 +216,8 @@ public void load() {
 		}
 
 		if (!isErrored) {
+			GFX.objectLabel(GFX.GL_PROGRAM, programId(), "PRO " + name);
```

Comment:
Comment said "needs to be set after binding the program". Assuming bind = use then it shouldn't be bound yet here?

Reply:
I see! Just ran it and everything works too. I'll mark this solved.